pkgs/games/hmcl/default.nix

```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   src = fetchurl {
     url = "https://github.com/huanghongxun/HMCL/releases/download/release-${version}/HMCL-${version}.jar";
-    hash = "sha256-bXZF38pd8I8cReuDNrZzDj1hp1Crk+P26JNiikUCg4g=";
+    hash = "sha256-ziqcauetWoFn58kBJ0KnqX5CPNC/Sn7DD/Buxdi977I=";
   };
```

Comment:
It's been like that since the first commit that added HMCL: https://github.com/NixOS/nixpkgs/commit/b4395db6756b2872fadbd8060d35a8a2e1f23f5b

We could try switching to build from source, although that is out of scope for this PR which is supposed to be a simple version bump.

Reply:
Upon further inspection, upstream uses Gradle, which is... [really](https://github.com/NixOS/nixpkgs/blob/075849637086d5e68302705ae44a3d985044938f/pkgs/applications/networking/instant-messengers/signald/default.nix#L41-L84) [notoriously](https://github.com/NixOS/nixpkgs/blob/c8b2579f1fe502a7cff41fbc55e8958f27650287/pkgs/by-name/ar/armitage/package.nix#L49-L66) [difficult](https://github.com/NixOS/nixpkgs/pull/310416) to make reproducible. Fetching prebuilt bytecode might be the only feasible option here.